plugins/out_stackdriver/stackdriver.c

```diff
@@ -761,21 +761,67 @@ static int get_stream(msgpack_object_map map)
     return STREAM_UNKNOWN;
 }
 
+static int make_bool_map(struct flb_stackdriver *ctx, msgpack_object *map,
+                         bool_map_t *bool_map, int map_num, 
+                         flb_sds_t to_be_removed[], int removed_num) {
+    msgpack_object_kv *kv;
+    msgpack_object *key;
+
+    char is_to_delete;
+    int i, j;
+    int ret = 0;
+    flb_sds_t removed;
+
+    for (i = 0; i < map_num; i++) {
+        bool_map[i] = TO_BE_REMAINED;
+    }
+
+    /* tail of map */
+    bool_map[map_num] = TAIL_OF_ARRAY;
+    if (map != NULL) {
+        kv = map->via.map.ptr;
+        for (i = 0; i < map_num; i++) {
+            key = &(kv + i)->key;
+            is_to_delete = FLB_FALSE;
+            if (key->type == MSGPACK_OBJECT_STR) {
```

Comment:
another line where we can remove one indentation level:

```c
if (key->type != MSGPACK_OBJECT_STR) {
    continue;
}

Reply:
updated